src/EnergyPlus/CMakeLists.txt

```diff
@@ -83,6 +83,8 @@ SET( SRC
   DXCoils.hh
   DXFEarClipping.cc
   DXFEarClipping.hh
+  Data/BaseboardData.hh
+  Data/BaseData.hh
```

Comment:
Adding these new files to break out the data from a single file into multiple files is the only significant point of review for this refactor. Let me know if this is the direction we want to move.